atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraServer.java

```diff
@@ -41,6 +42,7 @@ public interface CassandraServer {
     /**
      * The only proxy that will be used to reach the Cassandra host.
      * */
+    @DoNotLog
     @Value.Lazy
```

Comment:
Just for my understanding, why have we opted for `DoNotLog` in this case? On the internal AtlasDb proxy we have err'd on side of caution and made timelock IPs unsafe (although it was a rather contentious decision).

Reply:
Ah, a good question! This was autogenerated, and I suspect because of the `@Redacted` annotation. The reason for redaction here is linked to #6092 and relates to performance (not security) reasons, so I would regard this as a false positive.

Reply:
I changed to Unsafe, as it's noted that that is allowed